time stamp
int16 ticks_left
int16 ticks_right
